plugins/woocommerce-admin/client/products/sections/product-inventory-section/test/product-inventory-section.spec.tsx

```diff
@@ -0,0 +1,175 @@
+/**
+ * External dependencies
+ */
+import { Form } from '@woocommerce/components';
+import { Product } from '@woocommerce/data';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * Internal dependencies
+ */
+import { getAdminSetting } from '~/utils/admin-settings';
+import { ProductInventorySection } from '../';
+
+jest.mock( '@woocommerce/tracks', () => ( { recordEvent: jest.fn() } ) );
+jest.mock( '~/utils/admin-settings', () => ( {
+	getAdminSetting: jest.fn(),
+} ) );
+
+describe( 'ProductInventorySection', () => {
+	beforeEach( () => {
+		jest.clearAllMocks();
+		( getAdminSetting as jest.Mock ).mockImplementation(
+			( key, value = false ) => {
+				const values = {
+					manageStock: 'yes',
+					notifyLowStockAmount: 5,
+				};
+				if ( values.hasOwnProperty( key ) ) {
+					return values[ key as keyof typeof values ];
+				}
+				return value;
+			}
+		);
+	} );
+
+	const product: Partial< Product > = {
+		id: 1,
+		name: 'Lorem',
+		slug: 'lorem',
+		manage_stock: false,
+	};
+
+	it( 'should render the sku field', () => {
+		render(
+			<Form initialValues={ product }>
+				<ProductInventorySection />
+			</Form>
+		);
+
+		expect(
+			screen.getByLabelText( 'SKU (Stock Keeping Unit)' )
+		).toBeInTheDocument();
+	} );
+
+	it( 'should disable the manage stock section if inventory management is turned off', () => {
+		( getAdminSetting as jest.Mock ).mockImplementation( ( key, value ) => {
+			const values = {
+				manageStock: 'no',
+			};
+			if ( values.hasOwnProperty( key ) ) {
+				return values[ key as keyof typeof values ];
+			}
+			return value;
+		} );
+
+		render(
+			<Form initialValues={ product }>
+				<ProductInventorySection />
+			</Form>
+		);
+
+		expect(
+			screen.getByText( 'Track quantity for this product' )
+				.previousSibling
+		).toHaveClass( 'is-disabled' );
+	} );
+
+	it( 'should not render the manage stock section if inventory management is turned on', () => {
```

Comment:
Should this actually be that the manage stock is not disabled?

Reply:
It should!  Thanks for the catch!